src/database/migrations/20210408192059-auditing-feedback-fields.ts

```diff
@@ -0,0 +1,221 @@
+// TODO remove from actual migrations, DataTypes will be used
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import { QueryInterface, DataTypes } from 'sequelize';
+
+export default {
+  up: async (queryInterface: QueryInterface): Promise<void> => {
+    // Transactions are automatically use because a namespace is injected into sequelize when fetching configurations
+    await queryInterface.sequelize.transaction(async () => {
+
+    await queryInterface.createTable('topic_feedback', {
+        id: {
+            field: 'id',
+            type: DataTypes.INTEGER,
+            autoIncrement: true,
+            primaryKey: true,
+        },
+        topicId: {
+            field: 'topic_id',
+            type: DataTypes.INTEGER,
+            allowNull: false,
+            // Defaults to cascade when generating the db, no action on migrations
+            onUpdate: 'CASCADE',
+            onDelete: 'NO ACTION',
+            // Different from sequelize model, on models I do this with associations so I can use types
+            references: {
+                model: 'course_topic_content',
+                key: 'course_topic_content_id',
+            },
+        },
+        userId: {
+            field: 'user_id',
+            type: DataTypes.INTEGER
+        },
+        feedback: {
+            field: 'feedback',
```

Comment:
This should be topic_feedback_feedback

If we dion't keep it consistent it will get extremely confusing

The only ones that don't get table prefix are created at, updated at and foreign keys